example/vanilla-k8s-block-driver/vsphere.conf

```diff
@@ -7,3 +7,7 @@ user = "vcenter username"
 password = "vcenter password"
 port = "443"
 datacenters = "list of comma seperated datacenters where node VMs are present"
+
+[FeatureStatesConfig]
+name = "csi-feature-states"
```

Comment:
Can we add some comment over here to help users to know this is a config-map and not secret where this vsphere.conf is created.

we can also indicate this somewhere in the field name or section name.
 
for example - FeatureStatesConfig to FeatureStatesConfigMap

Reply:
Done